fsw/src/sample_app_version.h

```diff
@@ -38,14 +40,19 @@
 #define SAMPLE_APP_MINOR_VERSION 1  /*!< @brief Minor version number. */
 #define SAMPLE_APP_REVISION      99 /*!< @brief Revision version number. Value of 99 indicates a development version.*/
```

Comment:
REVISION should be 0 instead of MISSION_REV being 00. Can use to set version string to development vs release.